lib/extensions/liveness_probe/liveness_probe.js

```diff
@@ -6,22 +6,21 @@ const Controller = require('../../controllers/base');
 const { logRequest } = require('../../logger');
 
 class LivenessProbe extends Controller {
-
   /**
    * Part of middleware mechanism, every middleware needs this static method adhering to this contract.
-   * 
+   *
    * @param {*} options - to check for availability of this extension
    * @reutrns {bool} whether this class is enabled if `options` indicate as much
    */
-  static isEnabled(options) {
-    return options.extensions 
-      && options.extensions.liveness_probe 
-      && options.extensions.liveness_probe.enabled;
+  static isEnabled (options) {
+    return options.extensions &&
+    options.extensions.liveness_probe &&
+    options.extensions.liveness_probe.enabled;
   }
```

Comment:
I much prefer the use of the optional chaining operator.  IIRC, we'll need to deprecate Node 12 now, instead of April, when it is EOL.

Reply:
Ah right, we'd need to deprecate node.js 12 then. But I agree that we should do that in any case, since current LTS is 16, and at least 14 should be available to pretty much everyone on any platform by now.

Reply:
OK, thought it was a TS only thing, but maybe my local node is just too old.  Will do.